requirements-travis.txt

```diff
@@ -14,7 +14,7 @@ mock==1.2.0
 aexpect==1.0.0
 psutil==3.1.1
 # stevedore for loading "new style" plugins
-stevedore==1.8.0
+stevedore>=1.8.0
```

Comment:
We should pin requirements to exact version here, to minimize changes and reduce false positives.  Suggestion:

```
stevedore==1.20.0
```

Which I tested locally and *seems* to work.  Let's try it on travis.